projects/kit/components/input-inline/input-inline.component.ts

```diff
@@ -79,6 +79,13 @@ export class TuiInputInlineComponent
 
     onFocused(focused: boolean): void {
         this.updateFocused(focused);
+
+        if (focused) {
```

Comment:
This is not enough, unfortunately. We need to update scroll position of the wrapper to be in sync with the input, and input scroll position can change whenever. Try listening to `selectionchange` event, if it doesn't help — we probably would have to use polling with rAF when input is focused. Fucking Safari piece of shit 🤬 We can probably isolate it into a service/host directive with early return when not in Safari.

Reply:
No, that's need for another PR
This is reproduced in chrome when input has too long value

Reply:
Ok

Reply:
Then remove "closes" from this PR as it only addresses part of the issue